CONTRIBUTING.rst

```diff
@@ -114,3 +114,7 @@ Tips
 To run a subset of tests::
 
     $ python -m unittest tests.beprof
+
+
+Some classes have main methods which can be used for testing. To run them either use PyCharm right click on __main__ or
+use PYTHONPATH=. hack.
```

Comment:
It is not clear what is `PYTHONPATH=. hack` Provide some copy-pasteable example like:

`PYTHONPATH=. python beprof/curve.py`

Mention also that it applies only when trying to run the code after downloading it to local disc via git.

And a question - will this one work also ?

`python -m beprof.curve`